index.ts

```diff
@@ -479,12 +484,23 @@ addTests('isUserSettings', [
 	'isRepliesSettings',
 ]);
 
-export const isRepoTree = (url: URL | HTMLAnchorElement | Location = location): boolean => isRepoRoot(url) || Boolean(getRepo(url)?.path.startsWith('tree/'));
+export const isRepoTree = (url: URL | HTMLAnchorElement | Location = location): boolean => isRepoRoot(url)
+	|| Boolean(getRepo(url)?.path.startsWith('tree/'))
+	|| isRepoTreeFileFinder(url);
```

Comment:
This causes the url `GitHub.com?search=hi` to be `isRepoTree`

Reply:
Sorry, I don't understand, can you clarify?

Because:

* `https://github.com?search=hi` can **not** match `isRepoTree()` now and [c0e80db](https://github.com/refined-github/github-url-detection/pull/178/commits/c0e80db31948fe26c9d1731331ac47102a8c1618).
* Or, `https://github.com/sindresorhus/refined-github?search=hi` can match `isRepoTree()` now and [c0e80db](https://github.com/refined-github/github-url-detection/pull/178/commits/c0e80db31948fe26c9d1731331ac47102a8c1618).

Reply:
At the time of this review, with `https://github.com?search=hi`:

- `isRepoRoot(url)`: false because it has `search`
- `Boolean(getRepo(url)?.path.startsWith('tree/'))`: false because there's no `tree`
- `isRepoTreeFileFinder(url)` true because this function exclusively checked for the `search` parameter, regardless of where it appeared

so the result of `isRepoTree('https://github.com?search=hi')` was `true`

Reply:
I hope not to disturb you, I just want to clarify, if it causes you trouble, there is no need to answer.

---

> At the time of this review

Do you mean [c0e80db](https://github.com/refined-github/github-url-detection/pull/178/commits/c0e80db31948fe26c9d1731331ac47102a8c1618), and `https://github.com?search=1`? Because `isRepoTreeFileFinder` only match `?search` equals 1 ，for `?search=hi`, `isRepoTreeFileFinder` was `false`.

That is to say, for [c0e80db](https://github.com/refined-github/github-url-detection/pull/178/commits/c0e80db31948fe26c9d1731331ac47102a8c1618):

* isRepoTree('[https://github.com?search=hi](https://github.com/?search=hi)') was false
* isRepoTree('[https://github.com?search=hi](https://github.com/?search=1)') was true

But anyway, the current change is correct.

Reply:
Ah, yeah, I meant `search=1`